packages/components/src/rating/stories/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import Rating from '../';
+import '../style.scss';
```

Comment:
This broke my build, but removing this line got it up and running.

<img width="1155" alt="Screen Shot 2020-02-09 at 5 30 45 PM" src="https://user-images.githubusercontent.com/10561050/74113532-03273200-4b62-11ea-9f18-895e29b75c0b.png">

Reply:
We'll need to resolve this somehow. Without the styles pulled in, the component doesn't render properly:

<img width="211" alt="Screen Shot 2020-02-12 at 10 55 38 AM" src="https://user-images.githubusercontent.com/4167300/74283109-364a0b00-4d86-11ea-9d4c-6e376bbbf229.png">

Is there a top-level styles file (for all components) that we should pull into the storybook preview? Any other ideas how we can solve this?

Reply:
Aha! There is a mega style file for all components, now fixed here: ef17c27a42feb4f1045de37df7c740833576cc91